SPECS/liburing/liburing.spec

```diff
@@ -1,16 +1,15 @@
 Summary:        Linux-native io_uring I/O access library
 Name:           liburing
-Version:        2.0
-Release:        3%{?dist}
+Version:        2.3
+Release:        1%{?dist}
 License:        (GPLv2 WITH exceptions AND LGPLv2+) OR MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://git.kernel.dk/cgit/liburing/
 Source0:        https://brick.kernel.dk/snaps/%{name}-%{version}.tar.gz
 Patch1:         0001-spec-bump-version-to-2.0.patch
```

Comment:
The patches fails to apply as per the local testing I tried.

What worked is dropping all the patches; 

Digging a little into it reveals:
- Patch1 [not needed - we are already 2.3]
- Patch2 [not needed - it was introduced in 2.1]
- Patch3 [not-needed - it was introduced in 2.1 & reverted in 2.2]